recipes/rmlui/4.x/conanfile.py

```diff
@@ -0,0 +1,170 @@
+        }
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        
+        def lazy_lt_semver(v1, v2):
+            lv1 = [int(v) for v in v1.split(".")]
+            lv2 = [int(v) for v in v2.split(".")]
+            min_length = min(len(lv1), len(lv2))
+            return lv1[:min_length] < lv2[:min_length]
+
+        min_version = self._minimum_compilers_version.get(
+            str(self.settings.compiler))
+        if not min_version:
+            self.output.warn("{} recipe lacks information about the {} compiler support.".format(
+                self.name, self.settings.compiler))
+        else:
+            if lazy_lt_semver(str(self.settings.compiler.version), min_version):
+                raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
+                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+
+    def requirements(self):
+        if self.options.font_interface == "freetype":
+            self.requires("freetype/2.10.1")
+
+        if self.options.with_lua_bindings:
+            self.requires("lua/5.3.5")
+
+        if self.options.with_thirdparty_containers:
+            self.requires("robin-hood-hashing/3.9.1")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if not hasattr(self, "_cmake"):
+            self._cmake = CMake(self)
+            self._cmake.definitions["BUILD_LUA_BINDINGS"] = self.options.with_lua_bindings
+            self._cmake.definitions["BUILD_SAMPLES"] = False
+            self._cmake.definitions["CUSTOM_CONFIGURATION"] = True
+            self._cmake.definitions["CUSTOM_INCLUDE_DIRS"] = ";".join(self.deps_cpp_info["robin-hood-hashing"].include_paths)
+            self._cmake.definitions["DISABLE_RTTI_AND_EXCEPTIONS"] = not self.options.enable_rtti_and_exceptions
+            self._cmake.definitions["ENABLE_PRECOMPILED_HEADERS"] = True
+            self._cmake.definitions["ENABLE_TRACY_PROFILING"] = False
+            self._cmake.definitions["MATRIX_ROW_MAJOR"] = self.options.matrix_mode == "row_major"
+            self._cmake.definitions["NO_FONT_INTERFACE_DEFAULT"] = self.options.font_interface is None
+            self._cmake.definitions["NO_THIRDPARTY_CONTAINERS"] = not self.options.with_thirdparty_containers
+
+            self._cmake.configure()
```

Comment:
Theres a preference for our of source builds but it's not required just a note for the next one

Reply:
Would you please pinpoint the location of the material that can be used as reference in the future?